Dockerfile.ubi

```diff
@@ -180,6 +180,10 @@ RUN --mount=type=cache,target=/root/.cache/pip \
     --mount=type=cache,target=/root/.cache/uv \
     uv pip install https://github.com/flashinfer-ai/flashinfer/releases/download/v0.1.2/flashinfer-0.1.2+cu121torch2.4-cp311-cp311-linux_x86_64.whl
 
+# Install peft for prompt adapter support
```

Comment:
Sorry if I get back to this, but I just realized this is something which is only supported in the adapter, right? In that case this should be a dependency in `vllm-tgis-adapter` and not something installed here.

Reply:
This is something that is needed by vllm explicitly - https://github.com/opendatahub-io/vllm/blob/7a21f52109eb5102fc146b9025279ef4459d6435/vllm/prompt_adapter/models.py#L93

Reply:
Also see these lines:

Reply:
I can confirm that this is needed to run the upstream unit tests.